Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM golang:1.4
 
+RUN apt-get update && \
+    apt-get install -y librados-dev && \
+    rm -rf /var/lib/apt/lists/*
+
 ENV DISTRIBUTION_DIR /go/src/github.com/docker/distribution
 ENV GOPATH $DISTRIBUTION_DIR/Godeps/_workspace:$GOPATH
+ENV DOCKER_BUILDTAGS include_rados
```

Comment:
Will this blank out existing build tags env var? Should we be appending? For example, will this still work if overridden in the cli?